### PR TITLE
larger machine size that can fit default istio install

### PR DESCRIPTION
**Pull request by djazayeri**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by djazayeri** [comment]
Updated to use the tip directive.

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 1]
```diff
@@ -20,11 +20,18 @@ Follow these instructions to prepare an GKE cluster for Istio.
...
     $ gcloud container clusters create <cluster-name> \
       --cluster-version latest \
+      --machine-type=n1-standard-2 \
       --num-nodes 4 \
       --zone <zone> \
       --project <project-id>
     {{< /text >}}
 
+    {{< tip >}}
+    The default installation of mixer requires nodes with >1 vCPU. If you are doing a demo
```
Comment (navinger) [inline content/en/docs/setup/platform-setup/gke/index.md]:
```suggestion
    The default installation of Mixer requires nodes with >1 vCPU. If you are
```

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 2]
```diff
@@ -20,11 +20,18 @@ Follow these instructions to prepare an GKE cluster for Istio.
...
+    installation you may remove the `--machine-type` argument to use the smaller
```
Comment (navinger) [inline content/en/docs/setup/platform-setup/gke/index.md]:
```suggestion
    installing with the
    [demo configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
```

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 3]
```diff
@@ -20,11 +20,18 @@ Follow these instructions to prepare an GKE cluster for Istio.
...
+    `n1-standard-1` machine size instead.
```
Comment (navinger) [inline content/en/docs/setup/platform-setup/gke/index.md]:
```suggestion
     you can remove the `--machine-type` argument to use the smaller `n1-standard-1` machine size instead.
```

**Comment by rcaballeromx** [comment]
CLA overridden because both contributors are Googlers

**Comment by navinger** [comment]
The linter is complaining about a broken link. This doc: /docs/setup/kubernetes/additional-setup/config-profiles/

Got moved to:  /docs/setup/additional-setup/config-profiles/

**Comment by navinger** [comment]
@djazayeri, the linter is complaining about trailing whitespace. The 1.3 release is in the final stages. It would be great to get your change in. If you have a moment, please take a look. You can run `sudo make lint` in the root directory of your local fork.

**Pull request merged**